build/artifacts/rhel.Dockerfile

```diff
@@ -10,19 +10,30 @@
 #
 
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
-FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.11.13-11 as builder
-ENV PATH=/opt/rh/go-toolset-1.11/root/usr/bin:$PATH \
+FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.12.12-4 as builder
+ENV PATH=/opt/rh/go-toolset-1.12/root/usr/bin:$PATH \
     GOPATH=/go/
 USER root
 WORKDIR /go/src/github.com/eclipse/che-plugin-broker/brokers/artifacts/cmd/
 COPY . /go/src/github.com/eclipse/che-plugin-broker/
 RUN adduser appuser && \
+    dnf -y clean all && \
+    rm -rf /var/cache/yum && \
```

Comment:
These lines won't decrease image size afaik, since any installation/caching happened in an earlier layer.